### PR TITLE
Translate supported locales

### PR DESCRIPTION
**Pull request by filafb**
**What problem is this solving?**

This PR allows user to translate languages listed as supported locales.

**How should this be manually tested?**

This branch is linked in this [workspace](https://translsupportedlocales--vtexspain.myvtex.com/admin). Vtexspain has only one binding with default locale as `es-ES`, but has other languages as supported locales. With this code, it's possible to translate them all.

**Screenshots or example usage:**

Screen Shot 2021-04-19 at 4 52 03 PM

### CODE REVIEW

**Comment by cesarocampov** [comment]
@filafb we should think of changing the button UI because if the client has plenty locales as Muji, it will be supper cluttered.

**Comment by filafb** [comment]
> @filafb we should think of changing the button UI because if the client has plenty locales as Muji, it will be supper cluttered.

I will test the dropdown suggestion.

**Comment by filafb** [comment]
Marked as draft to work on the fields style position.

**Comment by filafb** [comment]
These new commits bring a new navigation interface as below. To achieve that, I had to uplift the export button to the reusable wrapper and pass its control as props to the children. 
1. [Button in Component Wrapper level](https://github.com/vtex-apps/admin-catalog-translations/pull/2/files#diff-e110dc44998c743a2c4a9030135614e4441044bca5f89cc17c513544ee68f6c6R29)
2. [Passing props to children component](https://github.com/vtex-apps/admin-catalog-translations/pull/2/files#diff-e110dc44998c743a2c4a9030135614e4441044bca5f89cc17c513544ee68f6c6R31-R35)

Screenshot_2021-04-29 Product

File: react/components/LocaleSelector.tsx [hunk 1]
```diff
@@ -76,21 +76,13 @@ const LocaleSelector: FC = () => {
...
-            </Button>
-          </div>
-        ))}
+    <div className="w-100 w4-ns">
+      <Dropdown
+        label="Available Language"
+        placeholder="Select a language"
+        value={selectedLocale}
+        options={convertToDropDownOptions(bindings)}
+        onChange={(_: unknown, value: string) => handleLocaleSelection(value)}
```
[the remark this answers is not in the document]
Comment (cesarocampov) [reply react/components/LocaleSelector.tsx:85]: we can have it on a new PR

**Review by acapeca — APPROVED** [review]

**Pull request merged**